app/jobs/calculate_score_overtime_job.rb

```diff
@@ -0,0 +1,55 @@
+class CalculateScoreOvertimeJob < ApplicationJob
+  queue_as :default
+
+  def perform test_id
+    # Do something later
+    corrected = []
+    test = Test.includes(:subject,
+                         test_questions: [:answers, {question: :answers}])
+               .find_by(id: test_id)
+    return unless test.present? && test.doing?
```

Comment:
em nên ... dùng toán tử an toàn như &. để check status của test nhé, chứ lỡ ... nó find by nil thì null pointer ngay đấy